### PR TITLE
Fixed definition of `place-content-*`

### PR DESCRIPTION
**Pull request by thecrypticace**
The utilities `place-content-between`, `place-content-around`, and `place-content-evenly` are defined incorrectly. This PR fixes them.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -7,7 +7,9 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
-- Nothing yet!
+### Fixed
+
+- Fixed definition of `place-content-between`, `place-content-around`, `place-content-evenly` ([#15440](https://github.com/tailwindlabs/tailwindcss/pull/15440))
```
Comment (adamwathan) [inline CHANGELOG.md:12]:
```suggestion
- Use the correct property value for `place-content-between`, `place-content-around`, and `place-content-evenly` utilities ([#15440](https://github.com/tailwindlabs/tailwindcss/pull/15440))
```

**Review by adamwathan — APPROVED** [review]

**Comment by jspm2013** [comment]
sorry guys but it seems (at least for me, just installed v4.0.0) that the value is still between instead of space-between, applying "place-content-between" on elements

**Comment by thecrypticace** [comment]
You sure you're not on an earlier version?

It looks right to me:

Screenshot 2025-01-25 at 13 27 32

**Comment by jspm2013** [comment]
aaargh...okay thank you @thecrypticace - even though I deleted and reinstalled node_modules and stuff before installing 4.0.0 it didn't work out, but yeah...user fault as usual - I forgot to also update tailwind/postcss, which was still on beta8... thank you for the great work!

**Pull request merged**